adapter/src/main/java/com/twilio/kudu/sql/KuduEnumerable.java

```diff
@@ -487,6 +488,7 @@ public Schema getTableSchema() {
     List<AsyncKuduScanner> scanners = predicates.stream().map(subScan -> {
       KuduScanToken.KuduScanTokenBuilder tokenBuilder = client.syncClient()
           .newScanTokenBuilder(calciteKuduTable.getKuduTable());
+      tokenBuilder.replicaSelection(ReplicaSelection.CLOSEST_REPLICA); // Added to avoid querying leader always
```

Comment:
how do we know that leader is being queried all the time?

Reply:
https://kudu.apache.org/apidocs/org/apache/kudu/client/AbstractKuduScannerBuilder.html 
```
Sets the replica selection mechanism for this scanner. The default is to read from the currently known leader.
```

The other option is to ensure we set `FaultTolerant` as true(which decreases throughput) and set READ_MODE as READ_AT_SNAPSHOT 
```
Fault tolerant scans use READ_AT_SNAPSHOT read mode. If no snapshot timestamp is provided, the server will pick one.
```

Reply:
Do we need to set replica selection to `ReplicaSelection.CLOSEST_REPLICA` if we set fault tolerant to true?
Might be good to keep the comment about fault tolerant has to be set to true if we want rows returned in the pk order.

Reply:
`ReplicaSelection.CLOSEST_REPLICA` will help us distribute our query traffic and is an independent change to the issue at hand.

Adding back the comment for rows returned in pk order.